tests/pr_test.go

```diff
@@ -86,6 +86,8 @@ func setupQuickstartOptions(t *testing.T, prefix string) *testschematic.TestSche
 		{Name: "prefix", Value: options.Prefix, DataType: "string"},
 		{Name: "existing_resource_group_name", Value: resourceGroup, DataType: "string"},
 		{Name: "region", Value: region, DataType: "string"},
+		{Name: "addons", Value: "{openshift-ai = {version = '417'}}", DataType: "string"},
```

Comment:
Can you please mention what all data type values you tried for which the test has failed? This will be helpful to avoid repeating the values that failed.

I remember you already tried `string` and `object({openshift-ai = optional(object(version = string))})`